packages/flutter/lib/src/material/slider.dart

```diff
@@ -406,14 +416,22 @@ class _RenderSlider extends RenderConstrainedBox implements SemanticsActionHandl
     }
   }
 
+  void _handleTapUp(TapUpDetails details) {
+    if (isInteractive && !_active)
+      onChanged(_discretize(_getValueFromGlobalPosition(details.globalPosition)));
+  }
+
   @override
   bool hitTestSelf(Point position) => true;
 
   @override
   void handleEvent(PointerEvent event, BoxHitTestEntry entry) {
     assert(debugHandleEvent(event, entry));
-    if (event is PointerDownEvent && isInteractive)
+    if (event is PointerDownEvent && isInteractive) {
+      // We need to add the drag first so that it has priority.
       _drag.addPointer(event);
+      _tap.addPointer(event);
+    }
```

Comment:
That's less flexible.  For example, there might be situations where you want to add a pointer to one recognizer but not the other (or maybe you want to add the pointer in a different order in some scenarios).